cmd/executor/cmd/root.go

```diff
@@ -200,7 +200,7 @@ func resolveDockerfilePath() error {
 // copy Dockerfile to /kaniko/Dockerfile so that if it's specified in the .dockerignore
 // it won't be copied into the image
 func copyDockerfile() error {
-	if _, err := util.CopyFile(opts.DockerfilePath, constants.DockerfilePath, ""); err != nil {
+	if _, err := util.CopyFile(opts.DockerfilePath, constants.DockerfilePath, "", -1, -1); err != nil {
```

Comment:
lets make the two integers into constants so that there meaning is more clear to readers

Reply:
done. There is now DoNotChangeUID and DoNotChangeGID.